LabSources/CustomRoles/Exchange2019/HostStart.ps1

```diff
@@ -304,6 +300,7 @@ function Start-ExchangeInstallation
     }
 }
 
+$exchangeDownloadLink = 'https://download.microsoft.com/download/d/7/b/d7bcf78a-00d2-4a46-a3d2-7d506116bcd2/ExchangeServer2019-x64-CU9.ISO'
```

Comment:
Hi @bill-long may I ask you to use `Get-LabConfigurationItem -Name Exchange2019Url` ? The URL then would be placed into AutomatedLab/AutomatedLab.init.ps1 like so:
```powershell
Set-PSFConfig -Module AutomatedLab -Name Exchange2019Url -Value 'https://download.microsoft.com/download/d/7/b/d7bcf78a-00d2-4a46-a3d2-7d506116bcd2/ExchangeServer2019-x64-CU9.ISO' -Validation String -Initialize -Description 'Exchange 2019 ISO URL'
```
This way, we can make it configurable :)

Reply:
In that case _all_ the Exchange custom roles should probably work that way. I've pushed the changes for that. Trying to figure out where AutomatedLab.init.ps1 lives so I can test this.

Reply:
Test is failing. Please do not merge until I figure out how I broke things with the latest commit.

Reply:
It's working. I just need to restart PowerShell to pick up the updated AutomatedLab.init.ps1.